src/wallet/rpcwallet.cpp

```diff
@@ -1346,9 +1346,13 @@ UniValue listtransactions(const JSONRPCRequest& request)
             "[\n"
             "  {\n"
             "    \"address\":\"address\",    (string) The bitcoin address of the transaction.\n"
-            "    \"category\":\"send|receive\", (string) The transaction category.\n"
-            "    \"amount\": x.xxx,          (numeric) The amount in " + CURRENCY_UNIT + ". This is negative for the 'send' category, and is positive\n"
-            "                                        for the 'receive' category,\n"
+            "    \"category\":\"send|receive|generate|immature|orphan\",     (string) The transaction category.\n"
```

Comment:
You could improve formatting by, for instance, following:
https://github.com/bitcoin/bitcoin/blob/ae32806ea20235c4542b1a5d4011df5ae267b58a/src/wallet/rpcwallet.cpp#L3017-L3023